### ISSUE TITLE
GetSystemMetadata method doesn't get replica information

### ISSUE THREAD

**Issue by taojing2002** [ropensci/datapack#140]
I used a piece of R code to get system metadata of an object which has three replicas.
The system metadata can be see from:
https://mn-sandbox-ucsb-1.test.dataone.org/knb/d1/mn/v2/meta/testReplicateJsonLD.1711738610440
```
pid <- "testReplicateJsonLD.1711738610440"
mn <- MNode("https://mn-sandbox-ucsb-1.test.dataone.org/knb/d1/mn/v2")
user <- "http://orcid.org/0000-0002-8916-564X"
sysmeta <- getSystemMetadata(mn, pid)
```
However, the replica section is missing from R even though it got the replication policy part:
<img width="1021" alt="Screen Shot 2024-04-04 at 10 24 00 AM" src="https://github.com/DataONEorg/rdataone/assets/8062988/9f44aa31-db37-4ccf-b69e-e4f1b38c39af">
<img width="1023" alt="Screen Shot 2024-04-04 at 10 24 21 AM" src="https://github.com/DataONEorg/rdataone/assets/8062988/64d21109-9e82-4233-9c27-0e5511e47f8b">

**Comment by jeanetteclark** [comment]
there is no field for `replica` within the SystemMetadata class, so we would need to add it and parse the information from the sysmeta so it fits within it, probably as a data.frame?

**Comment by mbjones** [comment]
Yeah, I omitted that replica field because, at the time we were discussing removing it from sysmeta altogether, and it's not client editable so I skipped it. Is it needed for editing, or just for completeness?

**Comment by taojing2002** [comment]
For completeness. I think it should show the replica information.
Otherwise, it may confuse users in some circumstances.
Also Metacat should enforce that clients can't edit the replica parts when
they call MN.updateSystemMetadata:
https://github.com/NCEAS/metacat/issues/1867
